.ci/run_cc_tests.sh

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017 Intel Corporation
```

Comment:
I'm guessing there will be more things at some point, make docker, deal with coverage data, .... May as well have one single script as our "API" and never change that.